uasurfer.go

```diff
@@ -205,11 +216,7 @@ func copyLower(b []byte, s string) bool {
 			return false
 		}
 
-		if 'A' <= c && c <= 'Z' {
-			c += 'a' - 'A'
-		}
-
-		b[j] = c
+		b[j] = c | 0x20
```

Comment:
This is only safe if the characters are between A and Z, for example it changes @, [, ] etc